app/src/main/java/com/example/golocal/fragments/ProfileFragment.java

```diff
@@ -48,12 +52,14 @@ public class ProfileFragment extends Fragment {
     private EditText etBio;
     private Button btFinishEditBio;
     private ImageButton ibEditProfileImage;
+    private Button btChooseFromGallery;
     private ParseUser user;
     private final String KEY_BIO = "bio";
     private final String KEY_IMAGE = "profileImage";
     private Context context;
     private final String TAG = "ProfileFragment";
     public static final int CAPTURE_IMAGE_ACTIVITY_REQUEST_CODE = 42;
+    public final static int PICK_PHOTO_CODE = 1046;
```

Comment:
```suggestion
    public static final int PICK_PHOTO_CODE = 1046;
```
Also, why 1046 and why public?

Reply:
In that case, could we make it `private`?

Reply:
In cases like this, we can add a comment in code to just explain that this is a random number